website/docs/d/tg_gateway.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "ibm"
+page_title: "IBM : tg_gateways"
+sidebar_current: "docs-ibm-datasource-tg-gateways"
+description: |-
+  Manages IBM Cloud Infrastructure Transit Gateway.
+---
+
+# ibm\_tg_gateway
+
+Import the details of an existing IBM Cloud Infrastructure transit gateway as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+
+## Example Usage
+
+```hcl
+
+resource "ibm_tg_gateway" "new_tg_gw"{
+name="transit-gateway-1"
+location="us-south"
+global=true
+resource_group="30951d2dff914dafb26455a88c0c0092"
+} 
+
+data "ibm_tg_gateway" "ds_tggateway" {
+    id=ibm_tg_gateway.new_tg_gw.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `id` - (Required, string) The id of the gateway.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `gateways` - List of all Transit gateways in the IBM Cloud Infrastructure.
```

Comment:
The attributes are not part of gateways(list).All are higher level attributes

Reply:
yeah..removed it.